tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -180,7 +180,7 @@ function kbs_k8s_delete() {
 	pushd "$COCO_KBS_DIR"
 	kubectl delete -k config/kubernetes/overlays
 	# Verify that coco-tenant resources were properly deleted
```

Comment:
Is it worth updating the comment to something like:
`# Verify that KBS namespace resources were properly deleted`
in case we end up changing the namespace and for clarity?

Reply:
@stevenhorsman changes applied